invenio_vocabularies/contrib/awards/mappings/v7/awards/award-v1.0.0.json

```diff
@@ -0,0 +1,84 @@
+{
+  "mappings": {
+    "dynamic": "strict",
+    "properties": {
+      "$schema": {
+        "type": "keyword",
+        "index": "false"
+      },
+      "created": {
+        "type": "date"
+      },
+      "id": {
+        "type": "keyword"
+      },
+      "updated": {
+        "type": "date"
+      },
+      "uuid": {
+        "type": "keyword"
+      },
+      "version_id": {
+        "type": "integer"
+      },
+      "identifiers" : {
+        "properties": {
+          "identifier" : {
+            "type" : "keyword"
+          },
+          "scheme" : {
+            "type" : "keyword"
+          }
+        }
+      },
+      "pid": {
+        "type": "object",
+        "properties": {
+          "pk": {
+            "type": "integer"
+          },
+          "pid_type": {
+            "type": "keyword"
+          },
+          "obj_type": {
+            "type": "keyword"
+          },
+          "status": {
+            "type": "keyword"
+          }
+        }
+      },
+      "title": {
+        "type": "object",
+        "dynamic": true,
+        "properties": {
+          "en": {
+            "type": "search_as_you_type"
+          }
+        }
+      },
+      "number": {
+        "type": "keyword"
+      },
+      "funder": {
+        "type": "object",
+        "properties": {
+          "@v": {
+            "type": "keyword"
+          },
+          "id": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword",
+            "fields": {
+              "suggest": {
+                "type": "search_as_you_type"
```

Comment:
minor: I'm not sure it is needed. It's what we discussed, will depend on the UI cases:
- Users look for funders,
- Then for an award inside a funder (in which case, with a "keyword" name we can just filter).